### PR TITLE
Add support for Symfony 4.0

### PR DESCRIPTION
**Pull request by duxet**

### CODE REVIEW

**Comment by mente** [comment]
Hey, thanks for contribution. Why did you close it though?

**Comment by duxet** [comment]
Hey. I can reopen it, but it fails tests due to some Composer version conflicts.

File: .travis.yml [hunk 1]
```diff
@@ -3,13 +3,24 @@ language: php
...
+    
+matrix:
+  exclude:
+  - php: 5.6
+    env: SYMFONY_VERSION=^4.0
+  - php: 7.0
+    env: SYMFONY_VERSION=^4.0
 
 install:
+  - composer self-update --snapshot
```
Comment (mente) [inline .travis.yml]: i would stick to stable version

**Comment by mente** [comment]
Let's bump `composer/composer` require-dev to `dev-master` which has https://github.com/composer/composer/pull/6788 . Later I will update it to stable (I hope)

**Comment by duxet** [comment]
ok, fixed :)

File: .travis.yml [hunk 2]
```diff
@@ -3,11 +3,21 @@ language: php
...
     - SYMFONY_VERSION=^2.3.3
     - SYMFONY_VERSION=^2.8.0
     - SYMFONY_VERSION=^3.0
+    - SYMFONY_VERSION=^4.0
+    
+matrix:
+  exclude:
+  - php: 5.6
+    env: SYMFONY_VERSION=^4.0
+  - php: 7.0
```
Comment (mente) [inline .travis.yml:19]: Doesn't symfony 4.* require php 7.0? Why did you exclude it?

**Comment by mente** [comment]
0.3.1 is released

File: .travis.yml (hunk 2 above)
Comment (duxet) [reply .travis.yml:19]: There was an issue with PHP 7.0 version on Travis:

```
Package symfony/framework-bundle at version ^4.0 has a PHP requirement incompatible with your PHP version (7.0.22.0) 
```

**Pull request merged**